gapic/src/main/com/google/gapid/views/TracerDialog.java

```diff
@@ -167,6 +167,8 @@ public static void showSaveTraceDialog(Shell shell, Models models) {
   }
 
   public static void showTracingDialog(Client client, Shell shell, Models models, Widgets widgets) {
+	// TODO: I believe somewhere around here is where we should output a message about which
+	//  ANGLE package is being used if we're in ANGLE trace mode
```

Comment:
Not sure exactly where to output a message that tells the user "enabling angle package <pkg_name> to trace OpenGL app <app_name> with Vulkan over ANGLE" or something like that. Also need to have those two names available when I output the message.